std_msgs/Header header

uint8 matrix_size
float32 block_size
float32[2] agent_pos
float32[2] goal_pos

float32[100] obstacle_matrix                 
float32[100] agent_matrix                 
float32[100] nei_goal_matrix                 
float32[100] goal_matrix                 

